lib/srv/sshserver_test.go

```diff
@@ -291,254 +290,6 @@ func (s *SrvSuite) TestAgentForward(c *C) {
 	c.Fatalf("expected socket to be closed, still could dial after 150 ms")
 }
 
-// TestExecCommandLocalPtyAndRemotePty tests executing a command where you have
```

Comment:
I moved the relevant tests into the integration package. I went a little overboard here originally in testing things that really were not all that relevant and/or covered elsewhere.